### ISSUE TITLE
Spotlight visualisation of Knowledge , Attitudes , Behaviours and Practices  Baseline Summaries 

### ISSUE THREAD

**Issue by akahure** [onaio/gisida#480]
In Spotlight, the client would like to visualize a baseline survey as attached in the file 

https://docs.google.com/spreadsheets/d/17KaqtCgmu-AGCC8N6B8gBE-BOLnFR-3fMQPEi_J91Kc/edit#gid=1836581231

preview below 
![Screenshot - 2020-07-16T114642 777](https://user-images.githubusercontent.com/43808025/87650268-0bf7e180-c75a-11ea-8a42-e95eab353310.png)

The columns flagged in green showing the % of respondents for Knowledge, Attitudes, Behaviours and Practices  Baseline Summaries 

In building the layer we have a situation where if we include all the responses we will end up having a long hover option appearing on the fill layer, on mouse hover. A sample of just two questions is shown below 

![Screenshot (101)](https://user-images.githubusercontent.com/43808025/87647417-06010100-c758-11ea-94da-bc8679abd276.png)

The screenshot only shows two categories of respondents and we have about another 25 to include which will be messy to show

The proposed implementation would be one of two options:

- We can highlight some priority responses and showcase them on the mouse hover like the image above and then once a user clicks a Province (fill layer) we can show further details in the profile view as what we have in maps.lotfa as shown below 

![Screenshot 2020-07-16 at 11 36 17](https://user-images.githubusercontent.com/43808025/87648837-9b9c9080-c758-11ea-84e0-a068e2df70ba.png)

The LOE for this would take about 3 to 5 days to implement (and my personal recommendation to showcase the baseline) 

**OR** 

- We could categorize the responses (in collaboration with the client) into pillars such as 'Demographics' 'Health' 'Education' on the navigation tab and when a user opens the layer it can be shown as a fill layer - something to what we had in LOTFA as well (below)

![Screenshot 2020-07-16 at 11 40 14](https://user-images.githubusercontent.com/43808025/87649572-241b3100-c759-11ea-8ede-7536e438be9f.png)

**Comment by alepietrobon** [comment]
i agree with your suggestion @akahure . A side-menu seems like the right way to provide a full snapshot of each district.  I hope you get resourcing for this :)

**Comment by akahure** [comment]
Based on a discussion with Erick, we should include all the indicator respondents as layers since we might need to compare what this looks like on the site. 
@Janemesi do you have the CSV complete and can you add it here for reference?

**Comment by Janemesi** [comment]
@akahure This file has the [gbv awareness](https://docs.google.com/spreadsheets/d/1v9nW_0kqCIiStbtdJmjYEp2H-fna1qJvg7-nLauMQYQ/edit?usp=sharing) information. This file has the [demographics]([url](https://docs.google.com/spreadsheets/d/1GcNsg2d5BgkOt8b7c6DpTjFwcyUvmyeyW07LN4Sa4tg/edit?usp=sharing)) information

**Comment by akahure** [comment]
@kahama94 confirm the structure of the file is okay before we commence implementation for the profile views

**Comment by kahummer** [comment]
@akahure We've discussed with @Janemesi the implementation based on what we have on lotfa and the following came up.
1. The profile view would be too long if we decide to show every question on it. An alternative would be to have demographics on the profile view then have layers for the other questions.
2. I do not have a concrete design for the profile views I'll try to design a mock-up before starting the implementation. If we can get a design I'd really appreciate it.
The implementation will commence once I'm done with dppsomalia. I'd say on Thursday latest next week 
cc: @ukanga
